tests/mypy_test.py

```diff
@@ -327,7 +328,12 @@ def main():
             if distribution == "SQLAlchemy":
                 continue  # Crashes
 
-            if not is_supported(distribution, major):
+            distribution_path = Path("stubs", distribution)
+
+            if not is_probably_stubs_folder(distribution_path):
```

Comment:
How will we catch it if somebody forgets to put a METADATA.toml in a new stubs folder?

Reply:
Any better after 83b05f3?